packages/fx-core/src/question/create.ts

```diff
@@ -351,21 +351,13 @@ export class CapabilityOptions {
     };
   }
   static bots(inputs?: Inputs): OptionItem[] {
-    return inputs !== undefined && getRuntime(inputs) === RuntimeOptions.DotNet().id
-      ? // currently no ai bot for dotnet
-        [
-          CapabilityOptions.basicBot(),
-          CapabilityOptions.notificationBot(),
-          CapabilityOptions.commandBot(),
-          CapabilityOptions.workflowBot(inputs),
-        ]
-      : [
-          CapabilityOptions.basicBot(),
-          CapabilityOptions.aiBot(),
-          CapabilityOptions.notificationBot(),
-          CapabilityOptions.commandBot(),
-          CapabilityOptions.workflowBot(inputs),
-        ];
+    return [
```

Comment:
Does this change immediately take effect on:
- VS? Requires UI change?
- CLI? Directly work?

Reply:
VS: VS toolkit extension use its own wizard logic. We need to create another pr in VS repo to see AI bot in scaffold wizard.
CLI: Cli will directly work after this pr only if user has set TEAMSFX_CLI_DOTNET feature flag in system environment.